tests/Behat/Resources/suites.yml

```diff
@@ -8,16 +8,36 @@ default:
                 - sylius.behat.context.setup.channel
                 - sylius.behat.context.setup.locale
                 - webgriffe_sylius_akeneo.behat.context.setup.queue
-
-                - webgriffe_sylius_akeneo.behat.context.cli.consume_command
+                - webgriffe_sylius_akeneo.behat.context.setup.akeneo
 
                 - webgriffe_sylius_akeneo.behat.context.db.product
-                - webgriffe_sylius_akeneo.behat.context.db.queue
+                - webgriffe_sylius_akeneo.behat.context.cli.import_command
                 - webgriffe_sylius_akeneo.behat.context.system.filesystem
 
             filters:
                 tags: "@importing_products && @cli"
 
+        ui_importing_products:
```

Comment:
In this file, all "enqueuing" should be replaced with "importing"

Reply:
Done, anyway, all these files will be reviewed with the #145.

Reply:
Ahhh ok